### PR TITLE
refactor: recon module

### PR DESCRIPTION
**Pull request by blackHatMonkey**
Refactor recon module to remove duplicate and unnecessary code.

### CODE REVIEW

File: wifiphisher/common/recon.py [hunk 1]
```diff
@@ -512,11 +337,89 @@ def get_sorted_access_points(self):
...
+        # check to see if encryption type is either WEP or OPEN
+        if not encryption_type:
+            if "privacy" in encryption_info:
+                encryption_type = "WEP"
+            else:
+                encryption_type = "OPEN"
+
+        if encryption_type != "WEP" and found_wps:
+            encryption_type += "/WPS"
+
```
Comment (anakin1028) [inline wifiphisher/common/recon.py]: we only need to add `WPS` one time in the security column.

File: wifiphisher/common/recon.py [hunk 2]
```diff
@@ -512,11 +337,89 @@ def get_sorted_access_points(self):
...
+        return encryption_type
```
[the remark this answers is not in the document]
Comment (blackHatMonkey) [reply wifiphisher/common/recon.py]: Good catch :+1: . It seems to be an indentation problem.

**Comment by blackHatMonkey** [comment]
@anakin1028 Pushed the fixes for the indention bug. Let me know if you find anything else that needs to be corrected :smile_cat:

File: wifiphisher/common/recon.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (anakin1028) [reply wifiphisher/common/recon.py]: looks good now:)

**Comment by anakin1028** [comment]
Can you squash so I can merge it?

**Comment by blackHatMonkey** [comment]
@anakin1028 You can just use rebase and merge button or if you really want to to squash it use the squash and merge :smile: .

**Comment by anakin1028** [comment]
Thanks @BlackHatMonkey!

**Pull request merged**